### DISCUSSION TITLE
Can't connect to remote instance on the Linux version?

### DISCUSSION THREAD

**Discussion by ghost** [transmission/transmission#5963] [Support]
I want to connect to my instance of Transmission running on my VPS. IT works with Windows, but on Linux the option to connect to a server is not there. Is this not possible on Linux?

**Comment by tearfur** [comment]
You're probably using the GTK client on Linux and qt client on Windows, AFAIK only the qt client has this feature right now.

You can also use the qt client on Linux.

**Reply by ghost** [reply]
That's it, thanks. Will the feature be implemented into the GTP client at some point?

**Reply by tearfur** [reply]
One of the core developers did mention this is on his wishlist https://github.com/transmission/transmission/discussions/2414#discussioncomment-1986135, but the GTK client has not been actively developed for quite a while, so there is no telling when it will be implemented.
